app.js

```diff
@@ -28,12 +29,19 @@ app.get("/api/articles/:article_id", getArticleByID);
 
 app.get("/api/articles/:article_id/comments", getCommentsByArticleID);
 
+app.post("/api/articles/:article_id/comments", addCommentByArticleID);
 
 app.use((err, req, res, next) => {
-    const msg = {msg: "Bad request"};
-    const statusCode = err.statusCode ?? 400;
+    let message = {msg: "Bad request"};
+    let statusCode = err.statusCode ?? 400;
 
-    res.status(statusCode).send(msg);
+    if (err.code === "23503")
+    {
+        statusCode  = 404;
+        message.msg = "Not found";
+    }
+
+    res.status(statusCode).send(message);
```

Comment:
You could separate out these into a custom error handler and a psql error handler